distribution/src/main/packaging/scripts/prerm

```diff
@@ -79,12 +79,6 @@ if [ "$REMOVE_SERVICE" = "true" ]; then
     if command -v update-rc.d >/dev/null; then
         update-rc.d elasticsearch remove >/dev/null || true
     fi
-
-    SCRIPTS_DIR="/etc/elasticsearch/scripts"
```

Comment:
Not sure if we should remove this from here since we may need to delete the directory still if the user has upgrade from an older version and the directory still exists